### ISSUE TITLE
Using this toggles auto brightness on

### ISSUE THREAD

**Issue by nhowardmit** [jschiefner/dimmer-tweak#4]
Turn auto brightness off
Dim via this tweak
Undim via this tweak
Auto brightness gets toggled on
User gets the sads

**Comment by jschiefner** [comment]
I don't disagree actually, i've read multiple comments where people were confused about this behaviour, i will look into it and change that in the next release. Thanks for your suggestions!

**Comment by jschiefner** [comment]
Hey, i made a new release (v1.1.0) to address this issue. I hope the Auto-Brightness toggling is less intrusive now, would you mind giving me your thoughts on the new release?

**Comment by nhowardmit** [comment]
Finally got around to installing (and reinstalling...) and...

**Comment by nhowardmit** [comment]
Appears screenshot doesn’t make it onto here...

after upgrade no longer works

Opening settings generates an error:

AdvancedBrightnessSliderPreferences: stat() failed with error=45

“ReinstallEd” and got same error

removed, installed again, still same error

**Comment by jschiefner** [comment]
Normally you should be able to upload screenshots here. Where does this error appear exactly, in the preference pane of the tweak?

**Comment by nhowardmit** [comment]
Yes. Tweak is off. Open preferences and get that error. I can type up the trace back if need be

**Comment by nhowardmit** [comment]
Note:
Iphone SE(2) iOS 13.5

**Comment by jschiefner** [comment]
Hey, it seems like something changed in the Build System when i got a new MacBook. Could you to install those two .deb files manually and tell me if one or both work? I packed them up in a .zip file here:

[Archive.zip](https://github.com/jschiefner/advanced-brightness-slider-tweak/files/7848985/Archive.zip)

**Comment by nhowardmit** [comment]
I installed 11.0.3 and it works again

**Comment by jschiefner** [comment]
Could you try the following versions as well? These are compiled with a M1 Macbook which i will use from now on to release so it is crucial that they work as well.

[advancedbrightnessslider_1.1.0_m1_macbook_older_xcode_versions.zip](https://github.com/jschiefner/advanced-brightness-slider-tweak/files/7874858/advancedbrightnessslider_1.1.0_m1_macbook_older_xcode_versions.zip)

**Comment by jiyangzwb** [comment]
如果我关闭了AdvancedBrightnessSlider ，或者卸载了， 手机的根据环境光源自动调节亮度不能正常运行。

**Comment by jiyangzwb** [comment]
If turned off or uninstalled, the phone's automatic brightness adjustment function based on ambient light will not resume

**Comment by jschiefner** [comment]
@jiyangzwb can you specify what you tried to do, what you expected to happen and what happened instead?

**Comment by shMatrix** [comment]
Hello @jschiefner 
Thank you a lot for this really great tweak.
I would like to share with you my suggestions how your tweak can be improved more if you don't mind of cource. 
THB, currently I don't understand the logic of using auto brightness while adjusting brightness via your tweak: sometimes auto brightness starts to work right away once I increased or decreased brightness, sometimes not (i'm not talking about reduced white point range now). 
I would suggest to make it more clearly for example in that way:
Once brightness was adjusted via your tweak auto brightness mode should be completely disabled till user turns it on himself. And it could be cool to implement option like - "force auto brightness" directly in CC when it's needed, for example by touching the "sun icon" in brightness bar or in any other way. I understand that forcing auto brightness mode in reduced white point range could lead to unpredictable behavior, in that case I would suggest to implement logic\script like - once auto brightness was forced then tweak should disable "reduce white point" mode and force normal auto brightness mode.
What do you think?

**Comment by jschiefner** [comment]
Hi @shMatrix thank you so much for your thoughts, i greatly appreciate people taking interest in the tweak in this way 😊

I'm not too happy myself how this feature works but right now i don't have the time to work on this tweak. But I wanted to overhaul this feature for some time and add some kind of button like you suggested. I was gonna add it next to the "True Tone" button as a fourth one and maybe make that optional of some kind. When i have time again for this project i will for sure take your comment into consideration!
